.golangci.yml

```diff
@@ -0,0 +1,14 @@
+linters-settings:
+  wsl:
+    allow-cuddle-declarations: true
+
+issues:
+  exclude-rules:
+    - linters: ["gocritic"]
+      text: "ifElseChain:"
```

Comment:
I would also keep this in general and exclude when required. But i will leave it up to you.